### PR TITLE
Update dependencies (rustc nightly-2023-04-01, viper v-2023-03-29-1448)

### PR DESCRIPTION
**Pull request by viper-admin**
* [x] Update Viper version to `v-2023-03-29-1448`.
* [x] Update rustc version to `nightly-2023-04-01`.
* [x] Run `cargo audit` and fix the issues.
* [x] Manualy update outdated dependencies (see the list below).
* [x] Manualy run `cargo update`.

<details><summary>List of direct outdated dependencies:</summary>

```
$ mv prusti-contracts/Cargo.toml prusti-contracts/Cargo_disabled.toml
$ cargo outdated --root-deps-only --workspace

info: syncing channel updates for 'nightly-2023-04-01-x86_64-unknown-linux-gnu'
info: latest update on 2023-04-01, rust version 1.70.0-nightly (5e1d3299a 2023-03-31)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'llvm-tools'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustc-dev'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'llvm-tools'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustc-dev'
info: installing component 'rustfmt'
    Updating git repository `https://github.com/rust-lang/cargo.git`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_path_with_version/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_path_with_version/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_version_with_path/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_version_with_path/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_version_with_git/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_no_default_features/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/git_registry/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/deprecated_default_features/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_path_name/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_path_name/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_with_rename/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/merge_activated_features/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/merge_activated_features/in/primary`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/unknown_inherited_feature/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/unknown_inherited_feature/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_dev/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_dev/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_rename_with_rename_noop/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit_optional/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit_optional/in/primary`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_inherit_dependency/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_inherit_dependency/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_rename_with_rename/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/build_prefer_existing_version/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/deprecated_section/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/dev_prefer_existing_version/in/dependency`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_inherit_optional_noop/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_inherit_optional_noop/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_features/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_workspace_dep_features/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_workspace_dep_features/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_default_features_with_no_default_features/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_path/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_path/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/add-basic.in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/list_features_path_no_default/in/primary`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit_features/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/detect_workspace_inherit_features/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/registry/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/features_preserve/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/require_weak/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/manifest_path_package/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/manifest_path_package/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_path_dev/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_path_dev/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_inline_features/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_inferred_name/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_inferred_name/in/primary`
warning: skipping duplicate package `your-face` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_path_noop/in/dependency`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_overwrite_inherit_dependency/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_overwrite_inherit_dependency/in/primary`
warning: skipping duplicate package `optional-dep` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_inferred_name_conflicts_full_feature/in/optional`
warning: skipping duplicate package `your-face` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_inferred_name_conflicts_full_feature/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_inferred_name_conflicts_full_feature/in/primary`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_inherit_noop/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_inherit_noop/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_default_features/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/change_rename_target/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_optional_with_no_optional/in`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_normalized_name/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/path_normalized_name/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_optional/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_no_optional_with_optional/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_rename_inherit_dependency/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/invalid_key_rename_inherit_dependency/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_rename_with_no_rename/in`
warning: skipping duplicate package `your-face` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_name_dev_noop/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_git_with_path/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_git_with_path/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_no_optional/in`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/preserve_unsorted/in`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_workspace_dep/in/dependency`
warning: skipping duplicate package `bar` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/overwrite_workspace_dep/in/primary`
warning: skipping duplicate package `optional-dep` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/list_features_path/in/optional`
warning: skipping duplicate package `your-face` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/list_features_path/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/list_features_path/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture-dependency` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_name/in/dependency`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/workspace_name/in/primary`
warning: skipping duplicate package `cargo-list-test-fixture` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/cargo_add/preserve_sorted/in`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/bin_already_exists_implicit/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_git/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/ignores_failure_to_format_source/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/mercurial_autodetect/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/lib_already_exists_nosrc/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/bin_already_exists_implicit_nosrc/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/creates_library_when_instructed_and_has_bin_file/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/fossil_autodetect/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/bin_already_exists_explicit/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/inferred_bin_with_git/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/lib_already_exists_src/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_hg_ignore_exists/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/inferred_lib_with_git/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/explicit_bin_with_git/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/formats_source/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_git_ignore_exists/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/auto_git/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_hg/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/bin_already_exists_implicit_namesrc/out`
warning: skipping duplicate package `foo` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/git_autodetect/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_bin/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/git_ignore_exists_no_conflicting_entries/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/creates_binary_when_both_binlib_present/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/creates_binary_when_instructed_and_has_lib_file/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/pijul_autodetect/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/bin_already_exists_implicit_namenosrc/out`
warning: skipping duplicate package `case` found at `/home/runner/.cargo/git/checkouts/cargo-e7ff1db891893a9e/17f8088/tests/testsuite/init/simple_lib/out`
analysis
================
Name        Project  Compat  Latest   Kind    Platform
----        -------  ------  ------   ----    --------
serde       1.0.155  ---     1.0.159  Normal  ---
serde_json  1.0.94   ---     1.0.95   Normal  ---
syn         1.0.109  ---     2.0.12   Normal  ---

proc-macro-tracing
================
Name         Project  Compat  Latest  Kind    Platform
----         -------  ------  ------  ----    --------
proc-macro2  1.0.52   ---     1.0.54  Normal  ---
quote        1.0.24   ---     1.0.26  Normal  ---
syn          1.0.109  ---     2.0.12  Normal  ---

prusti-common
================
Name   Project  Compat  Latest   Kind    Platform
----   -------  ------  ------   ----    --------
serde  1.0.155  ---     1.0.159  Normal  ---

prusti-utils
================
Name     Project  Compat  Latest   Kind    Platform
----     -------  ------  ------   ----    --------
serde    1.0.155  ---     1.0.159  Normal  ---
toml     0.7.2    ---     0.7.3    Normal  ---
walkdir  2.3.2    ---     2.3.3    Normal  ---

viper
================
Name     Project  Compat  Latest   Kind    Platform
----     -------  ------  ------   ----    --------
futures  0.3.27   ---     0.3.28   Normal  ---
jni      0.20.0   ---     0.21.1   Normal  ---
serde    1.0.155  ---     1.0.159  Normal  ---
tokio    1.26.0   ---     1.27.0   Normal  ---

viper-sys
================
Name      Project  Compat  Latest  Kind    Platform
----      -------  ------  ------  ----    --------
jni       0.20.0   ---     0.21.1  Normal  ---
tempfile  3.4.0    ---     3.5.0   Build   ---

jni-gen
================
Name      Project  Compat  Latest  Kind    Platform
----      -------  ------  ------  ----    --------
jni       0.20.0   ---     0.21.1  Normal  ---
tempfile  3.4.0    ---     3.5.0   Normal  ---

vir
================
Name         Project  Compat  Latest   Kind    Platform
----         -------  ------  ------   ----    --------
proc-macro2  1.0.52   ---     1.0.54   Normal  ---
quote        1.0.24   ---     1.0.26   Normal  ---
regex        1.7.1    ---     1.7.3    Normal  ---
serde        1.0.155  ---     1.0.159  Normal  ---
syn          1.0.109  ---     2.0.12   Normal  ---
thiserror    1.0.39   ---     1.0.40   Normal  ---
walkdir      2.3.2    ---     2.3.3    Build   ---

vir-gen
================
Name         Project  Compat  Latest  Kind    Platform
----         -------  ------  ------  ----    --------
proc-macro2  1.0.52   ---     1.0.54  Normal  ---
quote        1.0.24   ---     1.0.26  Normal  ---
syn          1.0.109  ---     2.0.12  Normal  ---
tempfile     3.4.0    ---     3.5.0   Normal  ---

prusti-interface
================
Name   Project  Compat  Latest   Kind    Platform
----   -------  ------  ------   ----    --------
regex  1.7.1    ---     1.7.3    Normal  ---
serde  1.0.155  ---     1.0.159  Normal  ---

prusti-specs
================
Name         Project  Compat  Latest  Kind    Platform
----         -------  ------  ------  ----    --------
proc-macro2  1.0.52   ---     1.0.54  Normal  ---
quote        1.0.24   ---     1.0.26  Normal  ---
syn          1.0.109  ---     2.0.12  Normal  ---

prusti-viper
================
Name        Project  Compat  Latest   Kind    Platform
----        -------  ------  ------   ----    --------
regex       1.7.1    ---     1.7.3    Normal  ---
serde       1.0.155  ---     1.0.159  Normal  ---
serde_json  1.0.94   ---     1.0.95   Normal  ---

prusti-server
================
Name     Project  Compat  Latest   Kind    Platform
----     -------  ------  ------   ----    --------
clap     4.1.8    ---     4.2.1    Normal  ---
reqwest  0.11.14  ---     0.11.16  Normal  ---
serde    1.0.155  ---     1.0.159  Normal  ---
tokio    1.26.0   ---     1.27.0   Normal  ---
warp     0.3.3    ---     0.3.4    Normal  ---

prusti-smt-solver
================
Name     Project  Compat  Latest  Kind    Platform
----     -------  ------  ------  ----    --------
futures  0.3.27   ---     0.3.28  Normal  ---

test-crates
================
Name   Project  Compat  Latest   Kind    Platform
----   -------  ------  ------   ----    --------
clap   4.1.8    ---     4.2.1    Normal  ---
serde  1.0.155  ---     1.0.159  Normal  ---
toml   0.7.2    ---     0.7.3    Normal  ---

systest
================
Name      Project  Compat  Latest  Kind    Platform
----      -------  ------  ------  ----    --------
jni       0.20.0   ---     0.21.1  Normal  ---
tempfile  3.4.0    ---     3.5.0   Build   ---

$ mv prusti-contracts/Cargo_disabled.toml prusti-contracts/Cargo.toml
```
</details>

@Aurel300 could you take care of this?

### CODE REVIEW

**Comment by JonasAlaif** [comment]
What's the status on this @Aurel300, it's starting to become an issue.

**Comment by vakaras** [comment]
@Aurel300 Any updates on this?

**Comment by Aurel300** [comment]
@vakaras No, I haven't had the time to investigate the bitvector issues yet. (Also I don't know why there would be BV issues at all…)

File: prusti-server/src/process_verification.rs [hunk 1]
```diff
@@ -173,9 +173,6 @@ fn new_viper_verifier<'v, 't: 'v>(
         }
     } else {
         report_path = None;
-        if backend_config.backend == VerificationBackend::Silicon {
-            verifier_args.extend(vec!["--disableTempDirectory".to_string()]);
-        }
```
Comment (fpoli) [inline prusti-server/src/process_verification.rs:178]: Is this going to be reverted before merging the PR?

Comment (vakaras) [reply prusti-server/src/process_verification.rs:178]: No. The option is gone in the latest Viper.

**Pull request merged**